src/appshell/qml/dockwindow/DockFrame.qml

```diff
@@ -108,6 +108,8 @@ Rectangle {
         hoverEnabled: false
         propagateComposedEvents: true
         enabled: tabsPanel.visible
+        acceptedButtons: Qt.NoButton
```

Comment:
This line causes a problem: now the tabs cannot be dragged anymore. In this case, because you are re-using an existing MouseArea, mouse clicks / dragging should still be accepted, of course; so this `acceptedButtons` line is only necessary when adding a new MouseArea that should be exclusively for hovering and not for clicking/dragging.